third_party/libusb/naclport/src/chrome_usb/types.cc

```diff
@@ -254,6 +333,20 @@ void InterfaceDescriptorConverter::VisitFields(const InterfaceDescriptor& value,
   callback(&value.extra_data, "extra_data");
 }
 
+template <>
+StructValueDescriptor<ConfigDescriptor>::Description
+StructValueDescriptor<ConfigDescriptor>::GetDescription() {
+  return Describe("chrome_usb::ConfigDescriptor")
+      .WithField(&ConfigDescriptor::active, "active")
+      .WithField(&ConfigDescriptor::configuration_value, "configurationValue")
+      .WithField(&ConfigDescriptor::description, "description")
+      .WithField(&ConfigDescriptor::self_powered, "selfPowered")
+      .WithField(&ConfigDescriptor::remote_wakeup, "remoteWakeup")
+      .WithField(&ConfigDescriptor::max_power, "maxPower")
+      .WithField(&ConfigDescriptor::interfaces, "interfaces")
+      .WithField(&ConfigDescriptor::extra_data, "extra_data");
```

Comment:
ditto

Reply:
Same as above - it's dictated by the naming in Chrome Extensions API.